commands/saml.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"fmt"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+
+	"github.com/spf13/cobra"
+)
+
+var SamlCmd = &cobra.Command{
+	Use:   "saml",
+	Short: "SAML related utilities",
+}
+
+var SamlAuthDataReset = &cobra.Command{
+	Use:     "authdatareset",
+	Short:   "Reset AuthData field to Email",
+	Long:    "Resets the AuthData field for SAML users to their email. Run this utility after setting the 'id' SAML attribute to an empty value.",
+	Example: " saml authdatamigrate",
```

Comment:
Can you please use the new example format to elaborate on what things can be done with the command? A good example is the `mmctl user create` command